test/yulPhaser/GeneticAlgorithms.cpp

```diff
@@ -212,6 +212,8 @@ BOOST_FIXTURE_TEST_CASE(runNextRound_should_generate_individuals_in_the_crossove
 BOOST_AUTO_TEST_SUITE_END()
 BOOST_AUTO_TEST_SUITE(ClassicGeneticAlgorithmTest)
 
+// FIXME: These tests run *very* slowly (tens of seconds). Investigate, fix and re-enable them.
+/*
 BOOST_FIXTURE_TEST_CASE(runNextRound_should_select_individuals_with_probability_proportional_to_fitness, ClassicGeneticAlgorithmFixture)
```

Comment:
Does boost have something like a `DISABLED_` prefix (similar to gtest) which places these disabled from running but still compiling?

Reply:
Good point. Looks like there is. There's the `disable()` decorator. I changed the code to use it.

One downside of it is that disabled tests get re-enabled when you use filters on the command line, which I often do (e.g. `--run_test=Phaser`): [Re: [Boost-users] Boost Unit Test: How to run only the enabled tests of a test suite?](https://lists.boost.org/boost-users/2017/12/88215.php)
Fortunately these tests are not actually broken, just slow so in this case it's fine but it would not work if we wanted to completely disable them.